### DISCUSSION TITLE
I see warning beings reported with the latest versions of casadi (3.6.0 and above) with do-mpc 4.5.1

### DISCUSSION THREAD

**Discussion by dipankar-thermosphr** [do-mpc/do-mpc#370] [Q&A]
There are warning reported from casadi as I upgrade the casadi versions to the latest with my do-mpc 4.5.1. The warnings are triggered when I call simulator.setup(). I don't see this warning tillI use casadi 3.5.6 versions.

I have attached the screenshots of the warnings. Any ideas on that?
<img width="1309" alt="Screenshot 2023-04-26 at 16 59 21" src="https://user-images.githubusercontent.com/102789987/234617556-e6a529e0-ed8e-4122-8372-139e4c3f61c2.png">

**Comment by 4flixt** [comment]
Hi @dipankar-thermosphr ,

We flagged the issue in #364 and already have a fix ready in develop. A next release is coming soon. For now you can safely ignore the warning.

Best,
4flixt
